### PR TITLE
soju: init at 0.1.2

### PR DESCRIPTION
**Pull request by malte-v**
###### Motivation for this change
Add soju, a user-friendly IRC bouncer

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/soju/default.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  postInstall = ''
+    scdoc < doc/soju.1.scd > doc/soju.1
+    installManPage doc/soju.1
+  '';
+
+  meta = with lib; {
+    description = "A user-friendly IRC bouncer";
+    homepage = "https://soju.im";
+    license = licenses.agpl3Only;
+    maintainers = with maintainers; [ malvo ];
```
Comment (jonringer) [inline pkgs/applications/networking/soju/default.nix:36]: Please add yourself as a maintainer in `lib/maintainers` and make this into a separate commit:

```
maintainers: add <name>
```

Comment (jonringer) [reply pkgs/applications/networking/soju/default.nix:36]:
```
error: undefined variable 'malvo'

       at /home/jon/.cache/nixpkgs-review/pr-123815/nixpkgs/pkgs/applications/networking/soju/default.nix:36:39:

           35|     license = licenses.agpl3Only;
           36|     maintainers = with maintainers; [ malvo ];
             |                                       ^
           37|   };
```

Comment (malte-v) [reply pkgs/applications/networking/soju/default.nix:36]: Oops, thought I already did that ._.

**Review by jonringer — APPROVED** [review]
LGTM
shows usage

```
https://github.com/NixOS/nixpkgs/pull/123815

1 package built:
soju
```
